fix_includes.py

```diff
@@ -1906,7 +1906,8 @@ def _DecoratedMoveSpanLines(iwyu_record, file_lines, move_span_lines, flags):
     # to do the comment-replacing first.
     sort_key = firstline.line
     iwyu_version = iwyu_record.full_include_lines.get(m.group(1), '')
-    if _COMMENT_LINE_RE.search(iwyu_version):  # the iwyu version has comments
+    if _COMMENT_LINE_RE.search(iwyu_version) or (flags.update_comments == 'none'):  
```

Comment:
There's an 80-char line width limitation in python code too, so this overruns.

Luckily, python code usually doesn't use parentheses around `if` predicates, so if you remove the parens, it fits perfectly :)